test/generator_test.rb

```diff
@@ -1,30 +0,0 @@
-require_relative 'test_helper'
-
-module Generator
-  class UpdateVersionAndGenerateTestsTest < Minitest::Test
```

Comment:
This test is still required, or the `build_tests` call in `generator.rb` is never tested.
This test should be renamed though.

Reply:
Taking a peek now!